seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -326,8 +326,16 @@ angular.module('BE.seed.controller.inventory_list', [])
             },
             has_meters: function () {
               if ($scope.inventory_type === 'properties') {
-                var inventory_ids = $scope.selectedOrder.slice().reverse();
-                return inventory_service.properties_meters_exist(inventory_ids).then(function (has_meters) {
+                const selected_property_view_ids = _.map(
+                  _.filter($scope.gridApi.selection.getSelectedRows(), function (row) {
+                    return row.$$treeLevel === 0;
+                  }),
+                  'property_view_id'
+                ).reverse();
```

Comment:
This was easier than refactoring $scope.selectedOrder, which is tied up with other uses. This is a pattern I basically copied from line 722

Reply:
I think this is definitely a reasonable approach. The amount of effort needed to refactor the grid selections to use view_ids is definitely non-negligible, and it might be up for debate as to whether it should happen in the first place.